### PR TITLE
pythonPackages.dotfiles: 0.6.3 -> 0.6.4

### PR DESCRIPTION
**Pull request by tsaeger**
###### Motivation for this change

upgrade minor version

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by FRidh** [comment]
Thanks. This is an application, and not a Python library. Could you move the expression out of `python-packages.nix`.

**Comment by tsaeger** [comment]
Ok - moved 'dotfiles' out of pythonPackages as requested.

Kept the version update as separate commit.

**Comment by tsaeger** [comment]
*bump*

File: pkgs/applications/misc/dotfiles/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl, python27Packages }:
```
Comment (FRidh) [inline pkgs/applications/misc/dotfiles/default.nix]: Is explicitly python27 needed, or could it also potentially work with other versions of Python? We try to stay as generic as possible, so `pythonPackages` has preference, but if it doesn't work with 3.x, then `python2Packages` (or `python27Packages`, that doesn't really matter anymore) should be used.

File: pkgs/applications/misc/dotfiles/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  name = "dotfiles-${version}";
+  version = "0.6.4";
+
+  meta = with stdenv.lib; {
+    description = "Easily manage your dotfiles";
+    homepage = https://github.com/jbernard/dotfiles;
+    license = licenses.isc;
+  };
+
+  src = fetchurl {
```
Comment (FRidh) [inline pkgs/applications/misc/dotfiles/default.nix]: `fetchPypi`

File: pkgs/applications/misc/dotfiles/default.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    # https://github.com/jbernard/dotfiles/archive/v${version}.tar.gz
+    url = "mirror://pypi/d/dotfiles/${name}.tar.gz";
+    sha256 = "03qis6m9r2qh00sqbgwsm883s4bj1ibwpgk86yh4l235mdw8jywv";
+  };
+
+  buildInputs = with python27Packages; [ pytest ];
```
Comment (FRidh) [inline pkgs/applications/misc/dotfiles/default.nix]: `checkInputs` (note that his only a Python thing)

File: pkgs/applications/misc/dotfiles/default.nix (hunk 1 above)
Comment (tsaeger) [reply pkgs/applications/misc/dotfiles/default.nix]: done.

File: pkgs/applications/misc/dotfiles/default.nix (hunk 2 above)
Comment (tsaeger) [reply pkgs/applications/misc/dotfiles/default.nix]: done.

File: pkgs/applications/misc/dotfiles/default.nix (hunk 3 above)
Comment (tsaeger) [reply pkgs/applications/misc/dotfiles/default.nix]: done.

**Comment by FRidh** [comment]
Thanks. Pushed e1385f0a9461a1112c74f22f37ed3d56b5620770 and also to 17.09.

**Comment by tsaeger** [comment]
Thanks!

FYI - I don't yet see e1385f0a9, though I see the cherry-picked commit in 17.09

**Comment by FRidh** [comment]
Oh, forgot to push it. So, its d6cebc8b3942eda96ae527a30039c5ca9f6362c0 now.

**Pull request closed without merging**